api/ratelimit/config/ratelimit/v3/rls_conf.proto

```diff
@@ -0,0 +1,39 @@
+syntax = "proto3";
+
+package ratelimit.config.ratelimit.v3;
+
+option java_package = "io.envoyproxy.ratelimit.config.ratelimit.v3";
+option java_outer_classname = "RlsConfigProto";
+option java_multiple_files = true;
+option go_package = "github.com/envoyproxy/go-control-plane/ratelimit/config/ratelimit/v3;ratelimitv3";
+
+// [#protodoc-title: Rate Limit Config Discovery Service (RLS Conf DS)]
+
+// Rate-limit config model
+message RateLimitConfig {
+  string domain = 1;
+  repeated RateLimitDescriptor descriptors = 2;
+}
+
+// Rate-limit descriptor model
+message RateLimitDescriptor {
+  string key = 1;
+  string value = 2;
+  RateLimitPolicy rate_limit = 3;
+  repeated RateLimitDescriptor descriptors = 4;
+  bool shadow_mode = 5;
+}
+
+// Rate-limit policy model
+message RateLimitPolicy {
+  string unit = 1;
+  uint32 requests_per_unit = 2;
+  bool unlimited = 3;
+  string name = 4;
+  repeated RateLimitReplace replaces = 5;
+}
+
+// Rate-limit replace model
+message RateLimitReplace {
+  string name = 1;
+}
```

Comment:
What is this?

Reply:
Described this in the comments of the proto file. This is for the "replace" feature.
Feature https://github.com/envoyproxy/ratelimit/tree/0b2f4d5fb04bf55e1873e2c5e2bb28da67c0643f#replaces
Example: https://github.com/envoyproxy/ratelimit/tree/0b2f4d5fb04bf55e1873e2c5e2bb28da67c0643f#example-7